python_modules/libraries/dagster-dbt/dagster_dbt_tests/test_asset_checks.py

```diff
@@ -246,7 +235,7 @@ def my_dbt_assets(context: AssetExecutionContext, dbt: DbtCliResource):
 def test_materialize_no_selection(dbt_commands: List[List[str]]) -> None:
     result = _materialize_dbt_assets(dbt_commands, selection=None, raise_on_error=False)
     assert not result.success  # fail_tests_model fails
-    assert len(result.get_asset_materialization_events()) == 9
```

Comment:
This seemed wrong to begin with, since there are 10 expected materializations (4 seeds + 6 models). Updating the manifest resulted in this error, so fixing it.